runtime/src/main/java/com/graphaware/runtime/DatabaseRuntime.java

```diff
@@ -59,7 +63,11 @@ protected DatabaseRuntime(RuntimeConfiguration configuration, GraphDatabaseServi
 
         // Register to topology change events
         // In Community Edition this raises a ClassNotFoundException
-        this.topologyListenerAdapter = new TopologyListenerAdapter((GraphDatabaseAPI) database);
+        try {
+            this.topologyListenerAdapter = new TopologyListenerAdapter((GraphDatabaseAPI) database);
+        } catch (Exception exception) {
+            LOG.warn("Topology change adapter is not available on community edition");
```

Comment:
Can't we reduce the level, if the user uses community he shouldn't get "warned" that some parts of the framework don't have to be instantiated.